std_msgs/Header header
float32 l_edge
float32 r_edge
float32 l_distance
float32 r_distance
float32 c_distance
float32 c_edge
int32 points_num
